packages/app/client/src/components/layout/Header/Header.js

```diff
@@ -56,9 +61,9 @@ const styles = theme => ({
 })
 
 const Header = ({ logoutUser, auth, classes, location }) => {
-  if (!auth.isAuthenticated) {
-    return null
-  }
+  // if (!auth.isAuthenticated) {
+  //   return null
+  // }
```

Comment:
Can we delete instead of commenting out?

Reply:
Yes